client/xamarin.forms/iOS/packages.tt

```diff
@@ -4,10 +4,12 @@
 <#@parameter type="System.String" name="StoreSDKVersion"#>
 <?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="xamarinios10" />
-  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="xamarinios10" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="xamarinios10" />
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="xamarinios10" />
+  <package id="Microsoft.Bcl" version="1.1.9" targetFramework="xamarinios10" />
```

Comment:
This is the minimum required version for this project. If we want to change it back to 1.1.10 we can, but it isn't required.